hawc/apps/lit/serializers.py

```diff
@@ -224,7 +224,7 @@ def to_representation(self, instance):
         assessment_id = self.context["assessment"].id
         root = models.ReferenceFilterTag.get_assessment_root(assessment_id)
         tree = root.dump_bulk(root, keep_ids=True)
-        return {"tree": tree[0]["children"]}
+        return {"tree": tree[0].get("children")}
```

Comment:
If just the root tag w/o children then `children` wouldn't be set; this circumvents the KeyError that was thrown.

Reply:
changed to return an empty list instead of null `tree[0].get("children", [])`